testsuite/MDAnalysisTests/utils/test_distances.py

```diff
@@ -325,7 +355,7 @@ def test_selfdist(self):
 
     def test_distarray(self):
         from MDAnalysis.lib.distances import distance_array
-        from MDAnalysis.lib.distances import transform_StoR, transform_RtoS
+        from MDAnalysis.lib.distances import transform_StoR
```

Comment:
you likely have to change some code for that when the functions where called directly.